includes/shortcodes.php

```diff
@@ -106,12 +224,13 @@ function tsml_next_meetings($arguments)
     return '<h3>Types</h3><ul>' . implode($types) . '</ul>';
 });
 
-//output a react meeting finder widget https://github.com/code4recovery/tsml-ui
-function tsml_ui()
+/*output a react meeting finder widget https://github.com/code4recovery/tsml-ui
+ *examples: [tsml_ui], with parameters: [tsml_ui key="type" value="women"] */
+function tsml_ui($atts, $content = null)
```

Comment:
$content doesn't seem to be used. could `$atts` be `$arguments` for consistency with other shortcodes?

Reply:
$atts is consistent with all the WordPress examples. How about we change that one instance of $arguments to be consistent with $atts?

Reply:
Unused $content parm removed.

Reply:
i believe that `$arguments` is better than `$atts` because it's clearer and not colloquial. same with `$parms` - it should be `$params` or, better, `$types`

keeping our variable names clear helps developer cognitive load 🙏

Reply:
Sorry, but I read somewhere that there was an advantage to using $atts rather than something like $args because there is some behind-the-scene preprocessing that goes on.  I think it had todo with the output validation, but I can't remember the details. I can't find the article this morning, but if I come across it I'll send it on to you.